cpp/inc/bond/ext/grpc/bond_utils.h

```diff
@@ -22,6 +25,35 @@
 
 namespace bond { namespace ext { namespace gRPC { namespace detail
 {
+    inline grpc::ByteBuffer to_byte_buffer(const OutputBuffer& output)
+    {
+        struct Buffers
+            : boost::intrusive_ref_counter<Buffers>,
+              boost::container::small_vector<blob, 8>
+        {};
+
+        boost::intrusive_ptr<Buffers> buffers{ new Buffers };
+        output.GetBuffers(*buffers);
+
+        boost::container::small_vector<grpc::Slice, 8> slices;
+        slices.reserve(buffers->size());
+
+        for (blob& data : *buffers)
+        {
+            data = blob_own(std::move(data));
+
+            slices.emplace_back(
+                const_cast<void*>(data.data()),
```

Comment:
Can there be a comment for why the const cast? (E.g., to adhere to the slice API?)

Reply:
The C++ API does not need `const_cast` actually. Left over code from older C API use.